mklogo.sh

```diff
@@ -3,17 +3,17 @@
 # GMT 中文社区 logo
 #
 gmt begin gmt-china-logo pdf,jpg,PNG,eps
-	gmt coast -Rd -JI110/10c -Dc -A10000 -G230/230/230 -S160/175/217
-	gmt coast -ECN+g204/204/204 -EHK+g204/204/204 -ETW+g204/204/204 -EMO+g204/204/204
-	echo 110 -20 GMT/China | gmt text -F+f35p,19,0/0/100
+    gmt coast -Rd -JI110/10c -Dc -A10000 -G230/230/230 -S160/175/217
+    gmt plot CN-border-L1.gmt -W0.2p,red
```

Comment:
```suggestion
    gmt plot CN-border-L1.gmt -G204/204/204
```

Reply:
why？

Reply:
Because lines are difficult to see for small images.